### DISCUSSION TITLE
PR merged although CI tests failed

### DISCUSSION THREAD

**Discussion by volker-raschek** [renovatebot/renovate#24262] [Request Help] [answered]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, tell us what version of Renovate you run.

docker.io/renovate/renovate:36.71.0-slim

### If you're self-hosting Renovate, select which platform you are using.

github.com

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe the problem

I am using renovate and have configured it against gitea to update my repositories. Recently I upgraded to version 36.71.0. Since then renovate automatically merges PRs, even though the change via CI has led to errors.

Is there any configuration change regarding renovate version v32 and v36, that changes the behavior of renovate when automerge is enabled and the CI throws errors, with the suggested change?

Here is one pull request which was merged even the CI tests throw errors: https://git.cryptic.systems/volker.raschek/markdownlint-docker/pulls/29

### Relevant debug logs

Logs are already removed :(

### Have you created a minimal reproduction repository?

I have linked to a minimal reproduction in the description above

**Comment by viceice** [comment, marked as answer]
Yes, read the breaking changes section: https://github.com/renovatebot/renovate/releases/tag/36.0.0
You should update more often to not get so many major versions behind.

Renovate defaults to platform automerge since v36. So gitea is doing the merge after required status checks are passing.
Same happens on github, so you need required status checks on branch protections

And please correct the template, first you select github.com as platform but running renovate against gitea. This contradicts.

BTW: Remove the `-slim` suffix, it's deprecated and will be stopped from publishing soon.
